its/ruling/src/test/resources/expected/kotlin/intellij-rust/kotlin-S125.json

```diff
@@ -0,0 +1,8 @@
+{
+'kotlin-intellij-rust-project:sources/kotlin/intellij-rust/src/main/kotlin/org/rust/ide/intentions/IfLetToMatchIntention.kt':[
+51,
+],
+'kotlin-intellij-rust-project:sources/kotlin/intellij-rust/src/main/kotlin/org/rust/ide/template/RsContextType.kt':[
```

Comment:
Not really an FP but weird that it does not pick up the 2 previous examples as well

Reply:
It is because it doesn't meet the minimum threshold to satisfy `words > 2`. We've got to make the cutoff somewhere, admittedly a bit unfortunate example, though :shrug: